website/status_screen/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from status_screen import views
+
+urlpatterns = [
+    path("<shift:shift>/", views.StatusScreen.as_view(), name="status"),
```

Comment:
Maybe we should add a path with a venue, that automatically redirects to the currently open shift in that venue.